svc-lighthouse-api/src/test/java/gov/va/vro/abddataaccess/service/FhirClientTest.java

```diff
@@ -50,6 +48,31 @@
 @Slf4j
 @ActiveProfiles("test")
 class FhirClientTest {
+  @AllArgsConstructor
+  @Getter
+  private static class BundleInfo {
```

Comment:
I would normally question the need for this to be a private static inner class, but I am okay with it since this MAY be some testing setup that could be used elsewhere -- in which case this should be refactored into a testing utility class. Otherwise 👍